### DISCUSSION TITLE
Error reading cgns file in cgns view

### DISCUSSION THREAD

**Discussion by rcruzfardilha99** [CGNS/CGNS#361] [CGNStalk]
I am having troubling reading a cgns file in CGNSview and this error shows up:

The file n0012.cgns was written with a more recent version of the CGNS library. You must update your CGNS library before trying to read this file

When I open the CGNS view  the version of the library is the 3.4 but the version that I installed wss the 4.2.0

**Comment by MicK7** [comment]
Please do not create issues on github, we only track jira issues.
